### PR TITLE
Changed some documentation links as some were broken (pointing to old pages)

### PR DESCRIPTION
**Pull request by ifbyol**
We discovered some links in the code that were broken after the docs revamp. I'm updating some of the links I detected

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

File: cmd/push.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func Push(ctx context.Context) *cobra.Command {
 		Hidden: true,
 		Use:    "push [service]",
 		Short:  "Build, push and redeploy source code to the target app",
-		Args:   utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/0.10/reference/cli/#push"),
+		Args:   utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/reference/okteto-cli/"),
```
Comment (ifbyol) [inline cmd/push.go:64]: There is not docs page for the push command, so we send them to the okteto cli page

File: cmd/stack/destroy.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ func Destroy(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "destroy <name>",
 		Short: "Destroy a compose",
-		Args:  utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/0.10/reference/cli/#destroy-2"),
+		Args:  utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/reference/okteto-cli/"),
```
Comment (ifbyol) [inline cmd/stack/destroy.go:38]: There is not docs page for the push command, so we send them to the okteto cli page

File: cmd/up/up.go [hunk 3]
```diff
@@ -393,7 +393,7 @@ func Up(at analyticsTrackerInterface, ioCtrl *io.Controller, k8sLogger *io.K8sLo
 			if up.Manifest.Type == model.OktetoManifestType && !up.Manifest.IsV2 {
 				oktetoLog.Warning("okteto manifest v1 is deprecated and will be removed in okteto 3.0")
 				oktetoLog.Println(oktetoLog.BlueString(`    Follow this guide to upgrade to the new okteto manifest schema:
-    https://www.okteto.com/docs/reference/manifest-migration/`))
+    https://www.okteto.com/docs/1.15/reference/manifest-migration/`))
```
Comment (ifbyol) [inline cmd/up/up.go:396]: The version `1.15` is the latest one where we have a reference to the manifest migration

**Review by andreafalzetti — APPROVED** [review]

File: cmd/up/up.go (hunk 3 above)
Comment (ifbyol) [reply cmd/up/up.go]: I rolled back this change as we recovered the page in the docs

**Review by jLopezbarb — APPROVED** [review]

**Pull request merged**